views/templates/front/mailalerts-account-line.tpl

```diff
@@ -28,9 +28,10 @@
   {$mailAlert.name}
   <span>{$mailAlert.attributes_small}</span>
   <a href="#"
-     class="js-remove-email-alert"
+     title="{l s='Remove mail alert' d='Modules.Mailalerts.Shop'}"
+     class="js-remove-email-alert btn btn-link"
      rel="js-id-emailalerts-{$mailAlert.id_product|intval}-{$mailAlert.id_product_attribute|intval}"
      data-url="{url entity='module' name='ps_emailalerts' controller='actions' params=['process' => 'remove']}">
-    X
+    <i class="material-icons">delete</i>
```

Comment:
Are you sure material is available in PS 1.7.1? https://github.com/PrestaShop/ps_emailalerts/blob/master/ps_emailalerts.php#L68

Reply:
Sorry, I don't know, if it isn't, we can change it to &times; (×).
You know it?

Reply:
Yes

https://github.com/PrestaShop/PrestaShop/blob/b9805532d49594e40765542345936086c5b8441f/themes/classic/_dev/package.json#L23

https://github.com/PrestaShop/PrestaShop/blob/1.7.1.x/themes/classic/assets/css/theme.css

And search "Material Icons"

Reply:
Thanks :+1: